### DISCUSSION TITLE
Velocity at a single location

### DISCUSSION THREAD

**Discussion by benoitFoloppe** [NatLabRockies/floris#420] [Q&A] [answered]
Hello,

I am wondering if in the new version of Floris (V3), there is an equivalent function to `get_set_of_points` (in `floris_interface.py`) that was available before ? What I need is to be able to extract velocity values along a line or at a single location.

Best regards,

**Comment by paulf81** [comment]
Hi @benoitFoloppe , thank you for posting this.  I created an issue to confirm this functionality is there and add some examples, if you would like to follow progress on Issue #425

**Comment by rafmudaf** [comment, marked as answer]
Hi @benoitFoloppe please see #650. It has been merged into the develop branch and will be released with v3.4. You can see an example of using this new feature in the [Examples](https://github.com/NREL/floris/blob/develop/examples/) folder and described in the [docs](https://nrel.github.io/floris/examples.html).
